compress.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+#Colors
+NONE='\033[00m'
+RED='\033[01;31m'
+GREEN='\033[01;32m'
+YELLOW='\033[01;33m'
+PURPLE='\033[01;35m'
+CYAN='\033[01;36m'
+WHITE='\033[01;37m'
+BOLD='\033[1m'
+UNDERLINE='\033[4m'
+BLINK='\x1b[5m'
+
+#Functions for CHDMAN & Maxcso
+compressCHDMAN () {
+    for file in ./*.{iso,cue}; do
+    name="${file%.*}" #Remove extension
+    chdman createcd -i "$file" -o "$name".chd --force
+done
+}
+compressMaxcso () {
+    for file in ./*.{iso,cue}; do
+    maxcso "$file"
+done
+}
+
+#Check for .storageInternal and determine if ROMs are in internal or not
+useInternalStorage=false
+FILE=~/dragoonDoriseTools/.storageInternal
+if [ -f "$FILE" ]; then
+	useInternalStorage=true
+	storageLocation="shared/roms"
+else
+	useInternalStorage=false
+	storageLocation="external-1"
+
+fi
+
+#Pick the systems
+SYSTEMS=$(whiptail --title "ROM Compressor" --checklist "Please select the systems whose ROMs you want to compress" 20 78 15 \
```

Comment:
```suggestion
SYSTEMS=$(whiptail --title "ROM Compressor" --checklist "Please select the systems whose ROMs you want to compress" 20 78 5 \
```

To cleanup a bit of the extra highlighting below the options